b1ddi/resource_ipam_address.go

```diff
@@ -31,13 +28,21 @@ func resourceIpamsvcAddress() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 		Schema: map[string]*schema.Schema{
```

Comment:
`parent` and `range` are given as optional below, but they should be "read only". It is read only in the API